src/renderer/components/caption-studio/TextEditor.vue

```diff
@@ -154,6 +154,7 @@ export default {
     EventBus.$on('caption_reset', this.reset);
     EventBus.$on('json_errors', this.onJsonErrors);
     this.$refs.Quill?.quill?.on('text-change', this.onEdit);
+    //this.$refs.Quill?.$el.addEventListener('paste', catchPasteEvent);
```

Comment:
its fine now bc it already made it to the release, but in the future we should make sure to remove commented code like this or at least leave a comment explaining why it is still there!